rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -143,13 +140,17 @@ def check_origin(self, origin):
     rospy.init_node("rosbridge_websocket")
     signal(SIGINT, SIG_DFL)
 
+    # Authentication options
+    authentication_method = rospy.get_param('~authentication_method', None)
```

Comment:
We should call this paramenter ```authentication_methods``` (with an 's') to make it clearer that it supports more than one method.